board/views.py

```diff
@@ -22,11 +120,11 @@ def __init__(self, game_id):
                       'genres.name, release_dates, '
                       'platforms.name, '
                       'aggregated_rating, aggregated_rating_count, '
-                      'rating, rating_count ',
+                      'rating, rating_count, slug ',
             'filter[id][eq]': game_id
```

Comment:
This also should be moved to a wrapper since `Game` class should know nothing about how IGDB API is implemented.